Lib/test/test_ast.py

```diff
@@ -1005,6 +1005,11 @@ def test_literal_eval_malformed_dict_nodes(self):
         malformed = ast.Dict(keys=[ast.Constant(1)], values=[ast.Constant(2), ast.Constant(3)])
         self.assertRaises(ValueError, ast.literal_eval, malformed)
 
+    def test_literal_eval_trailing_ws(self):
+        self.assertEqual(ast.literal_eval("    -1"), -1)
+        self.assertEqual(ast.literal_eval("\t\t-1"), -1)
```

Comment:
It would be better to test mixed tabs and spaces: `"  \t-1"` and `"\t  -1"`. It would fail if the code is `node_or_string.lstrip(" ").lstrip("\t")` or `node_or_string.lstrip("\t").lstrip(" ")`.

Reply:
Good idea, guess we can simply use ` \t -1`